lib/fastlane/plugin/wpmreleasetoolkit/actions/common/prototype_build_details_comment_action.rb

```diff
@@ -0,0 +1,259 @@
+
+           - The `app_center_app_name`, `app_center_release_id` and installation URL to use for the QR code to point to that release in App Center
+           - The `app_icon`
+           - The app's Build Number / versionCode
+           - The app's Version / versionName
+           - The app's Bundle ID / Application ID
+           - A `footnote` mentioning the MC tool for Automatticians to add themselves to App Center
+
+          This means that if you are using App Center to distribute your Prototype Build, the only parameters you *have* to provide
+          to this action are `app_display_name` and `app_center_org_name`; plus, for `metadata` most of the interesting values will already be pre-filled.
+
+          Any of those implicit default values/metadata can of course be overridden by passing an explicit value to the appropriate parameter(s).
+        DESC
+      end
+
+      def self.available_options
+        app_center_auto = '(will be automatically extracted from `lane_context if you used `appcenter_upload` to distribute your Prototype build)'
+        [
+          FastlaneCore::ConfigItem.new(
+            key: :app_display_name,
+            env_name: 'FL_PROTOTYPE_BUILD_DETAILS_COMMENT_APP_DISPLAY_NAME',
+            description: 'The display name to use for the app in the comment message',
+            optional: false,
+            type: String
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :app_center_org_name,
+            env_name: 'APPCENTER_OWNER_NAME', # Intentionally the same as the one used by the `appcenter_upload` action
+            description: 'The name of the organization in App Center (if you used `appcenter_upload` to distribute your Prototype build)',
+            type: String,
+            optional: true
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :app_center_app_name,
+            env_name: 'APPCENTER_APP_NAME', # Intentionally the same as the one used by the `appcenter_upload` action
+            description: "The name of the app in App Center #{app_center_auto}",
+            type: String,
+            optional: true,
+            default_value_dynamic: true # As it will be extracted from the `lane_context`` if you used `appcenter_upload``
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :app_center_release_id,
+            env_name: 'APPCENTER_RELEASE_ID',
+            description: "The release ID/Number in App Center #{app_center_auto}",
+            type: String,
+            optional: true,
+            default_value_dynamic: true # As it will be extracted from the `lane_context`` if you used `appcenter_upload``
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :app_icon,
+            env_name: 'FL_PROTOTYPE_BUILD_DETAILS_COMMENT_APP_ICON',
+            description: "An `:emojicode:` or URL to use for the icon of the app in the message. #{app_center_auto}",
```

Comment:
TIL, [Emojicode is a programming language](https://www.emojicode.org/). A common name for the `:emoji:` (e.g. `: grinning_face:` =  😁) in the context of programming is "emoji short codes", while the official spec uses "[CLDR Short Name](https://unicode.org/emoji/format.html#col-name)".

However, neither applies here, truly, given the emojis are fetched from Buildkite 🤔. 

What do you think of this suggestion?

```suggestion
            description: "The name of an emoji from the https://github.com/buildkite/emojis list or the full image URL to use for the icon of the app in the message. #{app_center_auto}",
```